torch/nn/modules/module.py

```diff
@@ -907,6 +904,10 @@ def register_full_backward_hook(
         in :attr:`grad_input` and :attr:`grad_output` will be ``None`` for all non-Tensor
         arguments.
 
+        Note that this hook need to apply a Function to the inputs and outputs of the Module
```

Comment:
Same comments here. The phrase "views of each other," is also a little vague if you'd like to keep this wording.